libsplinter/src/admin/rest_api/actix/proposals.rs

```diff
@@ -85,17 +85,26 @@ fn list_proposals<PS: ProposalStore + 'static>(
         None => DEFAULT_LIMIT,
     };
 
-    let mut link = req.uri().path().to_string();
```

Comment:
Please update commit message to say `management_type` instead of `mgmt_type`

Reply:
Fixed